store/tikv/coprocessor.go

```diff
@@ -604,21 +605,22 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *Backoffer, task *copTask, ch
 			IsolationLevel: pbIsolationLevel(worker.req.IsolationLevel),
 			Priority:       kvPriorityToCommandPri(worker.req.Priority),
 			NotFillCache:   worker.req.NotFillCache,
+			HandleTime:     true,
```

Comment:
I feel like "EnableHandleTime" is better.

Reply:
This is defined in protobuf, hard to change.